### ISSUE TITLE
GUI installer

### ISSUE THREAD

**Issue by WombleWoo7547** [Botspot/pi-apps#1815]
### Cool ideas?

I love Pi Apps because I don't like apt. I think it might be good if when you are installing it has a graphical installer interface. Maybe for people who like the terminal look the can enable it in Pi Apps Settings? Also I would like it if you could change to a beta/pre-release channel.

**Comment by theofficialgman** [comment]
> I think it might be good if when you are installing it has a graphical installer interface. 

A new interface during app installation is coming soon (next week) with a new GUI. However, the terminal will still show as all of these scripts are bash scripts, which are terminal commands, you can not simply make a GUI out of this, its against how pi-apps functions.

> Also I would like it if you could change to a beta/pre-release channel.

There are no beta/pre-release pi-apps software. When PRs/Zips are finalized they are merged into pi-apps. We do not merge unless they are majority bug free. There is no beta/pre-release model that makes sense for pi-apps, the master branch is up to date and always the most bug free and stable.

**Comment by Crilum** [comment]
>Also I would like it if you could change to a beta/pre-release channel.

@WombleWoo7547 do you mean beta versions for *apps*, or a beta of *Pi-Apps*?
